src/main/java/net/atos/zac/app/zoeken/model/RESTZaakZoekObject.java

```diff
@@ -88,6 +87,6 @@ public class RESTZaakZoekObject extends AbstractRESTZoekObject {
 
     public RESTZaakRechten rechten;
 
-    public Map<RolType.OmschrijvingGeneriekEnum, List<String>> betrokkenen;
+    public Map<String, List<String>> betrokkenen;
```

Comment:
same here, why replace an enum with a string? Enums are much better in general because they are typesafe.